{"name": "test_demo_test_four", "status": "failed", "statusDetails": {"message": "AssertionError: True is not false", "trace": "self = <tests.demo_test.DemoTest testMethod=test_demo_test_four>\n\n    def test_demo_test_four(self):\n>       self.assertFalse(True)\nE       AssertionError: True is not false\n\ntests/demo_test.py:15: AssertionError"}, "start": 1675438335638, "stop": 1675438335638, "uuid": "ebf2125e-1a55-4a57-91a6-fd68da4bcdd8", "historyId": "0582c4e8f19bba548442e60a9af6a6dc", "testCaseId": "85de7ec6257b56e3d7a3022ed0291807", "fullName": "tests.demo_test.DemoTest#test_demo_test_four", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "demo_test"}, {"name": "subSuite", "value": "DemoTest"}, {"name": "host", "value": "Elviss-MBP.broadband"}, {"name": "thread", "value": "79165-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.demo_test"}]}